config/application.rb

```diff
@@ -21,7 +21,8 @@ module MyTAPP
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
-
+    # Set deadline for contract acceptance as 2 weeks
+    ENV["deadline"] = (2*7*24*60*60).to_s
```

Comment:
Hmm. This should probably be a constant in one of the model classes with a fallback default set there, too.